### DISCUSSION TITLE
Happy to collab! (full stack dev)

### DISCUSSION THREAD

**Discussion by zach-hopkins** [itsToggle/plex_debrid#212] [General]
Love the project. Would love to contribute my talents to the repo. I'm a python dev and full stack Javascript dev. Let me know what ideas you have to make this thing shine. Excited about this since debrid has always seemed a bit clunky with Kodi. Keep it up!

**Comment by itsToggle** [comment]
That's awesome! Thanks so much for the offer :) I don't professionally code, so the stuff I've written until now is probably not the prettiest. 
Right now I'm mostly fixing bugs with a new update in the experimental branch. I'll create some sort of roadmap for the issues we could tackle next!
